public/v4/apps/calendso.yml

```diff
@@ -0,0 +1,93 @@
+captainVersion: 4
+services:
+    # Paperless-ng
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-db
+        restart: always
+        ports:
+            - $$cap_prisma_studio_port:5555
+        environment:
+            DATABASE_URL: postgresql://$$cap_dbuser:$$cap_dbpass@srv-captain--$$cap_appname-db:5432/$$cap_dbname
+            GOOGLE_API_CREDENTIALS: $$cap_google_api_credentials
+            NEXTAUTH_URL: $$cap_appname.$$cap_root_domain
+            NEXT_TELEMETRY_DISABLED: $$cap_next_telemetry_disabled
+            NODE_TLS_REJECT_UNAUTHORIZED: $$cap_tls_reject_unauthorized
+        caproverExtra:
+            containerHttpPort: '3000'
+            dockerfileLines:
+                - FROM ctadeu/calendso:$$cap_app_version
+                - CMD sh -c "npx prisma db push --preview-feature && yarn dev"
```

Comment:
1- `npx prisma` results in the latest version being pulled at runtime. To lock the behavior we have to freeze the version here. `@version`

2- Is `yarn dev` the way to go here? Isn't it for developement?

Reply:
@Ak4zh any update on this?

Reply:
@sandoche A docker image option is now available at https://hub.docker.com/r/ctadeu/calendso